requirements.txt

```diff
@@ -17,3 +17,4 @@ sentencepiece>=0.1.95
 filelock  # comes with 3rd-party dependency
 pycocotools>=2.0.2 ; python_version >= "3.7"
 kornia>=0.5.0
+pytest-flake8
```

Comment:
this is in the test requirements already

```suggestion
```

Reply:
as a new contributor if you follow the contributions guidelines
https://github.com/PyTorchLightning/lightning-flash/blob/master/.github/CONTRIBUTING.md#test-cases

you will call `make test` which without `pytest-flake8` you won't succeed with detecting the CI failures.
even with that the CI calls another version of flake8 and raises more errors than the ones you get in local dev_env

/cc @aribornstein

Reply:
In that case, we should mention in CONTRIBUTING to do `pip install .[dev,test]` as we do in Lightning before running `make test`.

https://github.com/PyTorchLightning/pytorch-lightning/blob/55525031c65756c2e8574cfee5d386e584aecdca/setup.py#L36-L48

and iron out the inconsistencies between the CI and local environment

I strongly suggest leaving these changes for a separate PR

cc: @Borda